allowlist.conf

```diff
@@ -170,3 +170,14 @@ yahoo.com.vn
 yeah.net
 yepmail.net
 your-mail.com
+tmpbox.net
+moakt.cc
+disbox.net
+tmpmail.org
+tmpmail.net
+tmails.net
+disbox.org
+moakt.co
+moakt.ws
+tmail.ws
+bareed.ws
```

Comment:
Looking at [this site](https://www.moakt.com/), it appears that these domains are disposable.
If so, they should be added to `disposable_email_blocklist.conf`, not `allowlist.conf`.

Reply:
Good catch!